roadmap/implementers-guide/src/node/disputes/participation.md

```diff
@@ -0,0 +1,94 @@
+# Dispute Participation
+
+Tracks all open disputes in the active leaves set.
+
+## Remarks
+
+`UnsignedTransaction`s are OK to be used, since the inner
+element `CommittedCanddidateReceipt` is verifiable.
+
+## ToDo
+
+1. Specify what is being gossiped, what messages and what they must contain.
+
+
+## IO
+
+Inputs:1. Receive `DisputeParticipationMessage::Resolution`
+1. Delete all relevant data associated with this dispute
+
+
+* `DisputeParticipationMessage::Detection`
+* `DisputeParticipationMessage::Resolution`
+
+Outputs:
+
+* `AvailabilityRecoveryMessage::RecoverAvailableData`
+* `CandidateValidationMessage::ValidateFromChainState`
+* `...::Blacklist`
```

Comment:
I believe the ellipsis here counts as an unresolved TODO. I think this is blacklisting a particular parachain block, right? In that case, possibly Candidate Backing could maintain the blacklist.

Reply:
There is already an API on the 
```
	pub fn unsafe_revert(
		&mut self,
		n: NumberFor<Block>,
		blacklist: bool,
	) -> sp_blockchain::Result<NumberFor<Block>> {
```

so that logic has to be made persistent, but is already there.